releng/org.eclipse.passage.loc.aggregator/pom.xml

```diff
@@ -1,16 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-	Copyright (c) 2018, 2020 ArSysOp and others
-
-	This program and the accompanying materials are made available under the
-	terms of the Eclipse Public License 2.0 which is available at
-	https://www.eclipse.org/legal/epl-2.0/.
-
-	SPDX-License-Identifier: EPL-2.0
-
-	Contributors:
-		ArSysOp - initial API and implementation
--->
+<!-- Copyright (c) 2018, 2020 ArSysOp and others This program and the accompanying 
```

Comment:
It looks like format of the section is broken for some reason. Please, restore it back.